etc/adminhtml/di.xml

```diff
@@ -67,4 +67,19 @@
                 sortOrder="10"/>
     </type>
 	<preference for="Magento\Framework\Image\Adapter\Gd2" type="Experius\WysiwygDownloads\Image\Adapter\Gd2" />
+    <type name="Magento\MediaGalleryIntegration\Plugin\SaveImageInformation">
+        <arguments>
+            <argument name="imageExtensions" xsi:type="array">
+                <item name="pdf" xsi:type="string">pdf</item>
+            </argument>
+        </arguments>
+    </type>
+    <type name="Magento\MediaGalleryRenditions\Model\Queue\FetchRenditionPathsBatches">
+        <arguments>
+            <argument name="batchSize" xsi:type="number">100</argument>
```

Comment:
Hi @mhaagen85,

batchSize does not change from the original batchSize.
This line can be removed, as otherwise we would override other batchSize "overrides" with this module, whilst not changing it.

The xml merge of Magento should accept only the addition of "pdf" in the fileExtensions array argument.

Kind regards, Boris